src/cephadm/cephadm

```diff
@@ -2420,6 +2420,7 @@ def get_container_mounts(ctx, fsid, daemon_type, daemon_id,
             if not os.path.exists(selinux_folder):
                 os.makedirs(selinux_folder, mode=0o755)
             mounts[selinux_folder] = '/sys/fs/selinux:ro'
+        mounts['/'] = '/rootfs:ro'
```

Comment:
@sebastian-philipp is it enough? Are there other places where I should add this change?

Reply:
We have to make sure `ceph ceph-volume` also properly mounts this. According to 

https://github.com/ceph/ceph/blob/d009895a6b2147a8f6f5bb32add778acdc42d598/src/cephadm/cephadm#L4990

this is already the case, so lgtm